e3sm_diags/parameter/diurnal_cycle_parameter.py

```diff
@@ -4,8 +4,8 @@
 class DiurnalCycleParameter(CoreParameter):
     def __init__(self):
         super(DiurnalCycleParameter, self).__init__()
+        # Custom attributes
+        # =============================
+        self.normalize_amp_int = 0
         self.print_statements = False
         self.normalize_test_amp = False
-        self.ref_timeseries_input = False
-        self.test_timeseries_input = False
```

Comment:
Remove `self.ref_timeseries_input = False` and `self.test_timeseries_input = False` because they are both default to `False` in parent `CoreParameter` class.